### PR TITLE
Make qsc easier to use

### PR DESCRIPTION
**Pull request by shuffle2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,48 @@
 
 Qt SDK Creator - A tool for automatically downloading, building and stripping down Qt
 
-## Examples
```
Comment (spycrab) [inline README.md:5]: I'd rather not have the examples section removed outright.

**Review by spycrab — APPROVED** [review]
LGTM, other than this.

File: README.md (hunk 1 above)
Comment (shuffle2) [reply README.md:5]: I've restored that

**Comment by shuffle2** [comment]
I've also added another commit, which enables accessibility. I tested that it fixes https://bugs.dolphin-emu.org/issues/12014

**Pull request merged**